src/bin/add/args.rs

```diff
@@ -4,42 +4,103 @@ use cargo_edit::Dependency;
 use cargo_edit::{get_latest_dependency, CrateName};
 use semver;
 use std::path::PathBuf;
+use structopt::StructOpt;
 
 use crate::errors::*;
 
-#[derive(Debug, Deserialize)]
-/// Docopts input args.
+#[derive(Debug, StructOpt)]
+#[structopt(bin_name = "cargo")]
+pub enum ArgsWrap {
```

Comment:
To me `Command` sounds like a more descriptive name than `ArgsWrap`, what do you think?

Reply:
Done